src/realtime/synchronizing-custom-data.md

```diff
@@ -61,11 +61,13 @@ Reliable properties are good for things that you update once and that should be
 
 The last option is an optional argument that specifies if you would like a change event added to the model. When this is set to true, a C# event is added; it will fire when a property is changed locally or remotely. This is a useful signal to update your scene to match the model.
 
-Now that we have our model defined, it’s time to compile it. Go back to Unity and highlight the `ColorSyncModel.cs` file in your project. The inspector should look something like this:
+If you're using Unity 2021 or newer, Normcore will automatically generate the remaining code needed to use your model. Don't worry if you don't see it in your file; the extra code exists as temporary C# code generated when your project compiles. At this point you can skip ahead to the next section.
+
+If you're using an older version of Unity, you will need to manually compile the model. Go back to Unity and highlight the `ColorSyncModel.cs` file in your project. The inspector should look something like this:
 
 ![](./synchronizing-custom-data/model-inspector.png)
 
-Now that our class has a `[RealtimeModel]` attribute on it. Normcore will detect it and display the model inspector. If your inspector is not updating correctly, **make sure your Unity project has no compile errors. If Unity is unable to compile this class, it will be unable to detect the changes you’ve made to your model.**
```

Comment:
We still need the [RealtimeModel] attribute right? Do you think it's still worth calling it out here in case that's the reason folks aren't seeing the model inspector?

Reply:
Sure, we can keep it in. I only removed it because the tense felt awkward with this paragraph being sort of conditional on the Unity version.

Reply:
Reverted in b2eeff3f16f73936242ecc5048283ead6c0837d5